nodejs/week3/app.js

```diff
@@ -0,0 +1,50 @@
+import express from "express";
+import knex from "./database_client.js";
+const app = express();
+const port = process.env.PORT || 3000;
+
+app.use(express.json());
+
+const apiRouter = express.Router();
+app.use("/api", apiRouter);
+
+const contactsAPIRouter = express.Router();
+apiRouter.use("/contacts", contactsAPIRouter);
+
+contactsAPIRouter.get("/", async (req, res) => {
+  let query = knex.select("*").from("contacts");
+
+  // if ("sort" in req.query) {
```

Comment:
Let's get into the habit of not leaving commented code in the commit :)

Reply:
Sure, I will keep this in mind